File: socialuni-js/packages/socialuni-api-base/src/model/comment/CommentAddVO.ts
import CommentVO from './CommentVO'

export default class CommentAddVO {
  public content: string
  public talkId: string
  public commentId: string
  public replyCommentId: string

  constructor(content: string, talkId: string) {
    this.content = content
    this.talkId = talkId
  }

  public toComment(): CommentVO {
    const comment: CommentVO = new CommentVO()
    comment.content = this.content
    return comment
  }
}

File: socialuni-server/socialuni-community-sdk/src/main/java/com/socialuni/social/community/sdk/model/QO/comment/CommentAddVO.java
package com.socialuni.social.community.sdk.model.QO.comment;

import lombok.Data;

import javax.validation.constraints.NotNull;

/**
 * TODO〈一句话功能简述〉
 * TODO〈功能详细描述〉
 *
 * @author qinkaiyuan
 * @since TODO[起始版本号]
 */
@Data
public class CommentAddVO {
    private String content;
    @NotNull
    private Integer talkId;
    private Integer commentId;
    private Integer replyCommentId;
}
